IDE/Renesas/e2studio/RX72N/EnvisionKit/wolfssl_demo/wolfssl_demo.c

```diff
@@ -254,30 +300,26 @@ static void Tls_client_init(const char* cipherlist)
         return;
     }
     #endif
-
-
-    /* use specific cipher */
-    if (cipherlist != NULL && 
-        wolfSSL_CTX_set_cipher_list(client_ctx, cipherlist) != 
-                                                            WOLFSSL_SUCCESS) {
-        wolfSSL_CTX_free(client_ctx); client_ctx = NULL;
-        printf("client can't set cipher list");
-    }
 }
 
-static void Tls_client()
+static void Tls_client(void *pvParam)
 {
     #define BUFF_SIZE 256
     #define ADDR_SIZE 16
     int             ret;
+#if defined(TLS_MULTITHREAD_TEST)
+    BaseType_t xStatus;
+#endif
+    TestInfo* p = (TestInfo*)pvParam;
```

Comment:
Please check pvParam or p is not NULL before using them.

Reply:
Thanks. Fixed.